src/sql_lex.xrl

```diff
@@ -44,7 +44,7 @@ Rules.
 (\|[:{\[#]([^\|]*)+\|)                              : parse_json(TokenLine, TokenChars).
 
 % names
-[A-Za-z][A-Za-z0-9_@:#\$]*                          : match_any(TokenChars, TokenLen, TokenLine, ?TokenPatters).
+[A-Za-z][A-Za-z0-9_@\$~]*                           : match_any(TokenChars, TokenLen, TokenLine, ?TokenPatters).

```

Comment:
Aren't `:` and/or `#` valid name characters?

Reply:
`:`is not, '#' is but conflicts with JSONPath.

Reply:
How can `#` conflicting with JSONpath? Aren't all JSONpaths are wrapped with `|`?